components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsNashornGraphBuilderFactory.java

```diff
@@ -90,7 +89,7 @@ public ScriptEngine createEngine(AuthenticationContext authenticationContext) {
         SelectAcrFromFunction selectAcrFromFunction = new SelectAcrFromFunction();
```

Comment:
When we move ACR function to conditional function repo, we can remove this

Reply:
Removed all relevant Function bindings classes and tests